src/Microsoft.TemplateEngine.Edge/LocalizableStrings.resx

```diff
@@ -395,4 +398,8 @@ Template cache will be recreated on the next run.</value>
     <value>'IWorkloadsInfoProvider' component provided by host provided some duplicated workloads (duplicates: {0}). Duplicates will be skipped.</value>
     <comment>{0} is the list of duplicates</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="TemplatePackageManager_Warning_DetectedTemplatesIdentityConflict" xml:space="preserve">
+    <value>The template {0} has the identity conflict with an already existing template {1}. The template {1} will be overwritten.</value>
```

Comment:
this error message might be improved:
- include information which package the template is coming from (in case of managed package)
- in case of unmanaged package, include the display name of provider.

Reply:
done